.github/workflows/pythonpackage.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        PYTHON_VERSION: [3.6.10, 3.7.6, 3.8.2]
+        PYTHON_VERSION: [3.6.10, 3.7.7, 3.8.2]
```

Comment:
Better to just not specify it and instead use latest of each? (Ie 3.6/3.7/3.8 and no minor version$